lib/auth/clt.go

```diff
@@ -2100,6 +2102,36 @@ func (c *Client) CreateResetPasswordToken(ctx context.Context, req CreateResetPa
 	})
 }
 
+// GetAppServers gets all application servers.
+func (c *Client) GetAppServers(ctx context.Context, namespace string, opts ...services.MarshalOption) ([]types.Server, error) {
+	cfg, err := services.CollectOptions(opts)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	resp, err := c.APIClient.GetAppServers(ctx, namespace, cfg.SkipValidation)
+	if err != nil {
+		return nil, trail.FromGRPC(err)
+	}
+
+	return resp, nil
+}
+
+// GetDatabaseServers returns all registered database proxy servers.
+func (c *Client) GetDatabaseServers(ctx context.Context, namespace string, opts ...services.MarshalOption) ([]types.DatabaseServer, error) {
+	cfg, err := services.CollectOptions(opts)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	resp, err := c.APIClient.GetDatabaseServers(ctx, namespace, cfg.SkipValidation)
+	if err != nil {
+		return nil, trail.FromGRPC(err)
+	}
+
+	return resp, nil
+}
+
```

Comment:
These functions were missing before?

Reply:
I refactored `APIClient.Get(X)Servers` to not depend on `services.MarshalOption`. These were added to wrap those endpoints with the original function definition required by `ClientI`. This is similar to the change I made a while back for `CreateResetPassword`.

Reply:
Okay makes sense.